specification/common/index.bs

```diff
@@ -2693,6 +2693,18 @@ location given for the start of the spanning direction.
 
 <h4 id="the-slur-element">The <dfn element><code>slur</code></dfn> element</h4>
 <section dfn-for="slur">
+
+The <code>end</code> attribute specifies the <a>measure location</a> of the
```

Comment:
I have two suggestions here:

1. I think we need to add an equivalent to MusicXML's line-type attribute here to specify solid, dashed, or dotted lines. Unlike most of MusicXML's attributes for the slur element, this one feels semantic to me. Might we want to encode the semantics though instead of the end results? For instance, a dashed slur may be dashed because it happens one time only through a repeated section, or because it is an editorial addition.
2. I think we also need a way to specify the side for the endpoint of the slur, perhaps with an end-side attribute. By default it would be the same value as the side attribute. However in some cases the sides differ, such as slurs that cross staves in a keyboard part.

At some point we want to handle slurs with multiple inflection points. I think that is more of a presentation issue than a semantic issue so we would not need to add this now.

Reply:
More fundamentally, I do not think that a measure location is sufficient for indicating either the start or end point of a slur. This is not sufficiently precise for contrapuntal music with multiple voices in one part or staff. 

MusicXML has slurs specified at the note level. In MNX-Common I think the starting and end points need to be specified at either the event or note level.

This leads me to think that the current MNX-Common spec has the slur element in the wrong place. Instead of being a spanning direction, it seems more appropriate as liaison content, which use notes for both the starting and end points.

Perhaps we may want to close this pull request and add a new issue to discuss the more fundamental move of slurs from spanning direction to liaison content.